index.html

```diff
@@ -1246,334 +1098,787 @@ <h3>DID URL Content Metadata</h3>
-										<li><b>contentMetadata</b>: <code>«[ <code>resolved DID document metadata</code> ]»</code></li>
-									</ol>
-								</li>
-							</ol>
-						</li>
-						<li>If the value of the <b>accept</b> <var>input <a href="#did-resolution-options">DID resolution option</a></var>
-							is <b>text/uri-list</b>:
-							<ol class="algorithm">
-								<li>Return the following result:
-									<ol class="algorithm">
-										<li><b>dereferencingMetadata</b>: <code>«[ ... ]»</code></li>
-										<li><b>content</b>: <code>« selected service endpoint URLs »</code></li>
-										<li><b>contentMetadata</b>: <code>«[ "contentType" → "text/uri-list", ... ]»</code></li>
-									</ol>
-								</li>
-							</ol>
-						</li>
-						<li>Otherwise, return the following result:
-							<ol class="algorithm">
-								<li><b>dereferencingMetadata</b>: <var>error object</var> with type set to <a href="#REPRESENTATION_NOT_SUPPORTED">REPRESENTATION_NOT_SUPPORTED</a></li>
-								<li><b>content</b>: <code>null</code></li>
-								<li><b>contentMetadata</b>: <code>«[ ]»</code></li>
-							</ol>
+				<li>
+					Use the <code>type</code> of the selected
+					<code>service</code> to determine how the
+					<a>DID URL</a> path is used in assembling the URL of the
+					resource to be resolved. The algorithm for assembling this
+					URL MUST be defined in the corresponding service type
+					specification. A service type specification MAY appear in
+					the <a href="#did-resolution-service-types">DID Service
+					Types</a> section of this document, the applicable <a>DID
+					method</a> specification, or in the DID Document
+					Properties Extensions [[?DID-EXTENSIONS-PROPERTIES]].
+					<ol>
+						<li>
+							The <code>FileService</code> type is currently the
+							only service type defined in this specification. See
+							the <a href="#fileservice-service-type">FileService
+							Service Type</a> for its definition.
 						</li>
 					</ol>
 				</li>
+				<li>
+					Resolve the assembled URL to obtain the final resource.
+					<ol>
+						<li>If the resource cannot be retrieved, the resolver
+						MUST return a <strong>NOT FOUND</strong> error.</li>
+						<li>If the <a>DID Resolver</a> does not support the
+						protocol of the assembled URL, the <a>DID Resolver</a>
+						must return an error.</li>
+					</ol>
+					
+					<div class="note">
+						<p>
+						Resolving an assembled URL — particularly one that is a
+						DID — might result in a <dfn>resolution cycle</dfn>,
+						which is a set of steps that result in an infinite loop.
+						For example, a <a>service endpoint</a> might indirectly
+						point back through a sequence of resolutions to a
+						previously dereferenced identifier. A <a>DID
+						resolver</a> recursively resolving a <a>service
+						endpoint</a> is advised to detect and handle such a
+						cycle to prevent an infinite loop or resolution failure.
```

Comment:
```suggestion
						previously dereferenced identifier. A <a>DID
						resolver</a> is expected to detect and handle
						recursive cycles when resolving a <a>service
						endpoint</a>, to prevent an infinite loop or
						other resolution failure.
```